file/lib/src/sentry_file.dart

```diff
@@ -53,7 +53,7 @@ class SentryFile implements File {
   }
 
   @override
-  Future<File> create({bool recursive = false}) {
+  Future<File> create({bool recursive = false, bool exclusive = false}) {
```

Comment:
@marandaneto Is this correct to also introduce the `exclusive` param like you did for the `createSync` method?